.webpack/webpack.prod.js

```diff
@@ -1,22 +1,59 @@
 /* global __dirname module */
 
 /*
-This configuration should be used for production installs.
-It is the default webpack configuration.
+Production webpack configuration.
 */
-const path = require('path');
 const webpack = require('webpack');
 const { merge } = require('webpack-merge');
 
-const common = require('./webpack.common');
-const projectRootDir = path.resolve(__dirname, '..');
+const TerserPlugin = require('terser-webpack-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 
-module.exports = merge(common, {
+const base = require('./webpack.base');
+
+module.exports = merge(base('production'), {
+  devtool: false,
   mode: 'production',
   plugins: [
     new webpack.DefinePlugin({
-      __OPENMCT_ROOT_RELATIVE__: '""'
+      __OPENMCT_ROOT_RELATIVE__: '""',
+      'process.env.NODE_ENV': JSON.stringify('production')
     })
   ],
-  devtool: 'source-map'
+  // TODO (@evenstensberg): splitchunks when application is aligned towards it
+  optimization: {
+    runtimeChunk: false,
+    minimizer: [
+      new TerserPlugin({
```

Comment:
why these settings specifically instead of defaults?

Reply:
Could use defaults too, but fine tuning optimization is probably better

Reply:
Would you mind to explain some of the optimizations briefly and why you chose them?

Reply:
I think the props of the terser plugin is pretty much self-explainatory, but mainly it's small tweaks on minification details like threading, comments, optmization levels (think of it like C(gcc) -O <level>) that is more aggressive towards minification. If the minification strategy (compress: 3) is too high, it might lead to the application crashing.

I was hoping that closure compiler optimization soon would be available for webpack 5, since it is very good, but I think it's still wip

(https://github.com/webpack-contrib/closure-webpack-plugin/issues/161)